specification/elasticsan/resource-manager/readme.md

```diff
@@ -50,7 +50,7 @@ This is not used by Autorest itself.
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net-track2
-  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-python
```

Comment:
Is this intentional? If not, despite this change makes sesne, I suggest to revert it